homeassistant/components/group/light.py

```diff
@@ -172,8 +183,32 @@ def should_poll(self) -> bool:
         if ATTR_FLASH in kwargs:
             data[ATTR_FLASH] = kwargs[ATTR_FLASH]
 
-        await self.hass.services.async_call(
-            light.DOMAIN, light.SERVICE_TURN_ON, data, blocking=True)
+        if emulate_color_temp_entity_ids:
+            emulate_color_temp_data = data.copy()
+            temp_k = color_util.color_temperature_mired_to_kelvin(
+                emulate_color_temp_data[ATTR_COLOR_TEMP])
+            hs_color = color_util.color_temperature_to_hs(temp_k)
+            emulate_color_temp_data[ATTR_HS_COLOR] = hs_color
+            del emulate_color_temp_data[ATTR_COLOR_TEMP]
+            emulate_color_temp_data[ATTR_ENTITY_ID] = list(
```

Comment:
Why do you need to convert it to a list, it's already a list?

Reply:
Fixed.